distribution/docs/src/main/jdocs/content/_plugins/geocoder-plugin-contents.adoc

```diff
@@ -0,0 +1,33 @@
+:type: plugin
+:status: published
+:title: GeoCoder Plugin
+:link: _geocoder_plugin
+:plugintypes: preingest
+:summary: Populates the `Location.COUNTRY_CODE` attribute if the Metacard has an associated location.
+
+The GeoCoder Plugin is a pre-ingest plugin that is responsible for populating the Metacard's `Location.COUNTRY_CODE` attribute if the Metacard has an associated location.
+If there is a valid country code for the Metacard, it will be in ISO 3166-1 alpha-3 format.
+If the metacard's country code is already populated, the plugin will *not* override it.
+The GeoCoder relies on either the WebService or <<_offline_gazetteer_service,Offline Gazetteer>> to retrieve country code information.
```

Comment:
I did notice when I added the link that we don't have anything for configuring the webservice, only the offline. I'd prefer to have it in the `configuring` section and link to it here, much like the offline